### PR TITLE
fix: truncate long tokenId

### PR DESCRIPTION
**Pull request by sahar-fehri**
## **Description**

This PR adds a popover for NFT tokenIds that are very long.

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/26179?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

### **Before**

Screenshot 2024-07-29 at 09 49 25

### **After**

https://github.com/user-attachments/assets/eeeb1452-fda6-43aa-b7f7-4d72b41713f4

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by NidhiKJha — DISMISSED** [review]
LGTM

**Review by EtherWizard33 — DISMISSED** [review]
LGTM

File: ui/components/app/assets/nfts/nft-details/nft-detail-information-row.tsx [hunk 1]
```diff
@@ -57,7 +84,37 @@ const NftDetailInformationRow: React.FC<NftDetailInformationRowProps> = ({
...
+          {withPopover && fullValue ? (
+            <>
+              <Box
+                onMouseEnter={handleMouseEnter}
+                onMouseLeave={handleMouseLeave}
+                ref={setBoxRef}
+              >
+                {value}
+              </Box>
+              <Popover
```
Comment (bergeron) [inline ui/components/app/assets/nfts/nft-details/nft-detail-information-row.tsx:96]: Any reason the `<Tooltip>` component was not used?

Comment (sahar-fehri) [reply ui/components/app/assets/nfts/nft-details/nft-detail-information-row.tsx:96]: The designs [here](https://www.figma.com/design/TfVzSMJA8KwpWX8TTWQ2iO/Asset-list-and-details?node-id=4313-14410&m=dev) use popover and not tooltip

**Review by gambinish — APPROVED** [review]

**Review by bergeron — APPROVED** [review]

**Pull request merged**